README.md

```diff
@@ -1,7 +1,5 @@
 # Personal-portfolio-microverse
-This project is my personal portfolio development at the start of Microverse.
-
-Microverse already prepared template in figma for all students to replicate. This is also a milestone project so not everything was completed at a go.
+My personal portfolio project at Microversee
```

Comment:
- [ ] The main header of the README file should be a descriptive title not a description of the project, please 🙏🏽 remove this part from the beginning for professional documentation.